ui/packages/catalog-ui-search/src/main/webapp/component/search-form-interactions/search-form-interactions.view.js

```diff
@@ -122,6 +125,19 @@ module.exports =  Marionette.ItemView.extend({
                 type: type
             });
         },
+        handleShare: function() {
+            lightboxInstance.model.updateTitle('Query Template Sharing');
+            lightboxInstance.model.open();
+            lightboxInstance.lightboxContent.show(new QueryTemplateSharing({
+                model: this.model,
+                permissions: {
+                    'accessIndividuals': this.model.get('accessIndividuals'),
+                    'accessGroups': this.model.get('accessGroups')
+                },
+                modelId: this.model.get('id')
+            }));
+            this.handleClick();
```

Comment:
❓ None of the other options need to manually call `this.handleClick()`. Do the rest of those just not need it or is this specific case interrupting the call chain somehow?